README.md

```diff
@@ -81,6 +81,7 @@ find_package(easy_profiler REQUIRED)
 
 add_executable(app_for_profiling ${SOURCES})
 
+target_include_directories(app_for_profiling $<TARGET_PROPERTY:easy_profiler,INTERFACE_INCLUDE_DIRECTORIES>)
```

Comment:
@uykusuz this is redundant and unnecessary, as this is already done automatically by the `PUBLIC` keyword used here:

https://github.com/yse/easy_profiler/blob/c352f34c690310d382dd72e1d2fa0a90f811be33/easy_profiler_core/CMakeLists.txt#L175-L178

which will automatically include the path to every user targets that are linking `easy_profiler`.